src/test/java/posters/pageobjects/pages/checkout/NewShippingAddressPage.java

```diff
@@ -60,42 +59,42 @@ public void validateStructure()
         $("#formAddDelAddr").shouldBe(visible);
```

Comment:
@oomelianchuk Can we reuse the AddressForm at this place? und implement this functionality at one place?

Reply:
Done